call_tracking/models.py

```diff
@@ -1,6 +1,6 @@
 from django.db import models
 from django.db.models import Count
-from django.utils.encoding import python_2_unicode_compatible
+from six import python_2_unicode_compatible
```

Comment:
Since python2 is no longer supported, I think we could safely delete the usage of this decorator, WDYT?

Reply:
You are right, I'm removing that decorators.